src/components/Navbar.js

```diff
@@ -3,91 +3,86 @@ import Notification, { notify } from 'react-notify-toast';
 import { withRouter, Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import {
+  Collapse,
+  Navbar as Navigation,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  UncontrolledDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap';
 import { logoutAction } from '../redux/actions/auth/authActions';
 
 export class Navbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false,
+    };
+    this.toggle = this.toggle.bind(this);
```

Comment:
This is solved when you use arrow functions

Reply:
@Duncanian I have worked on this.